Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM php:7.2.13-cli-alpine
+
+ADD . ~/app
+
+WORKDIR ~/app
+
+RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');"
```

Comment:
FYI composer's site recommends a different solution for downloading composer programmatically, we can mess with that later though, not really that important

there's surely some example we can rip from laradock or whatever other robust full-featured docker env for php too